File: module6/A0622I1-CoffeeStore_FrontEnd/src/app/modal/IFeedbackDto.ts
/**
 * IFeedbackDto interface is used to collect data from the API
 *
 * @author TuLG
 * @version 1.0
 * @since 2023-06-13
 */


export interface IFeedbackDto {
  id: number;
  fb_id?: string;
  name?: string;
  email?: string;
  date?: string;
  content?: string;
  type?: string;
  rate?: string;
}

File: module6/CoffeeStore_BackEnd/spring_boot/src/main/java/com/codegym/backend/dto/IFeedbackDto.java
/**
 * com.codegym.backend.dto.FeedbackDto interface is used to collect data from the database
 *
 * @author TuLG
 * @version 1.0
 * @since 2023-06-13
 */

package com.codegym.backend.dto;

public interface IFeedbackDto {
    Integer getId();
    String getFb_id();
    String getName();
    String getEmail();
    String getType();
    String getDate();
    String getRate();
}

